ConstantineMaliza/capstone-2
# Blog route

#### What does this PR do?
Blog Route
#### Description of Task to be completed?
Auth user can create blog delete and view blog
#### How should this be manually tested?
- clone project
- use postman to check
- run  `npm run dev`
- blog require blog title body and image
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
